plugins/cairo-lang-macro-stable/src/lib.rs

```diff
@@ -1,24 +1,44 @@
 use crate::ffi::StableSlice;
-use std::ffi::{CStr, CString};
+use std::ffi::CStr;
 use std::num::NonZeroU8;
 use std::os::raw::c_char;
 
 pub mod ffi;
 
+/// An option.
+///
+/// This struct implements FFI-safe stable ABI.
+#[repr(C)]
+#[derive(Debug)]
+pub enum StableOption<T> {
+    None,
+    Some(T),
+}
+
```

Comment:
You're using this to store pointers. Can't you put `NonNull` there and use standard Option instead?

https://doc.rust-lang.org/stable/std/option/index.html#representation